testdata/logging/logforwarding/rsyslog/insecure/rsyslogserver_configmap.yaml

```diff
@@ -18,6 +18,11 @@ data:
       if $msg contains "namespace_name=openshift" or $msg contains "namespace_name=default" or $msg contains "namespace_name=kube" then /var/log/custom/infra-container.log
       if not ($msg contains "namespace_name=openshift" or $msg contains "namespace_name=default" or $msg contains "namespace_name=kube") then /var/log/custom/app-container.log
     }
-    :programname, contains, "journal.system" /var/log/custom/infra.log
-    :programname, contains, "k8s-audit.log" /var/log/custom/audit.log
-    :programname, contains, "openshift-audit.log" /var/log/custom/audit.log
\ No newline at end of file
+    :programname, !contains, "kubernetes.var.log.containers" /var/log/custom/infra.log
+
+    :msg, contains, "docker"{
```

Comment:
I'm not sure if these configurations will work or not when testing the legacy syslog forward, @gkarager did you test it?